test/consts/zz_generated_gateway.go

```diff
@@ -0,0 +1,8 @@
+// This file is generated by test/internal/cmd/generate-gateway-urls. DO NOT EDIT.
+
+package consts
+
+const (
+	GatewayCRDsKustomizeURL    = "github.com/kubernetes-sigs/gateway-api/config/crd/experimental?ref=v0.5.0"
+	GatewayConformanceTestsURL = "https://raw.githubusercontent.com/kubernetes-sigs/gateway-api/v0.5.0"
```

Comment:
Hm could we come up with a different name for this that would not mention conformance tests? I believe the URL is pretty generic, i.e. it's just a base (prefix) for URLS trying to get raw content from github. WDYT @mlavacca ?

Reply:
yep, I agree. I'm going to change  the name